src/components/timeline/OverflowEdgeIndicator.js

```diff
@@ -12,6 +12,7 @@ import './OverflowEdgeIndicator.css';
 type Props = {
   className: string,
   children: React.Node,
+  panelLayoutGeneration: number,
```

Comment:
Could you please add a comment in `componentDidUpdate` explaining that every prop change, and especially `panelLayoutGeneration`, will refresh the indicator status ? I want to avoid forgetting about `panelLayoutGeneration` in case we change something there in the future.